sycl/include/sycl/ext/oneapi/experimental/group_helpers_sorters.hpp

```diff
@@ -343,8 +343,7 @@ class group_sorter {
 
   static std::size_t memory_required(sycl::memory_scope scope,
                                      size_t range_size) {
-    return 2 * joint_sorter<>::template memory_required<T>(
-                   scope, range_size * ElementsPerWorkItem);
+    return 2 * range_size * ElementsPerWorkItem * sizeof(T) + alignof(T);
```

Comment:
How different is this from what we had before? `alignof(T)` (because of `2` multiplier)?

Reply:
For this place difference is:
`2 * range_size * ElementsPerWorkItem * sizeof(T) + 2 * alignof(T)`
vs
`2 * range_size * ElementsPerWorkItem * sizeof(T) + alignof(T)`

Added comments clarifying the returned size.